pkg/crd/crdloader/crdloader.go

```diff
@@ -154,8 +155,33 @@ func formatGVK(group, version, kind string) string {
 	return fmt.Sprintf("{%v, %v, %v}", group, version, kind)
 }
 
+// LoadCRDs returns the list of CRDs Config Connector manages that map to GCP
+// resources.
 func LoadCRDs() ([]apiextensions.CustomResourceDefinition, error) {
-	crdsRoot := repo.GetCRDsPath()
+	return loadCRDs(repo.GetCRDsPath())
+}
+
+// LoadAllCRDs returns the list of all the CRDs Config Connector manages,
```

Comment:
Is there any reason to call LoadCRDs instead of LoadAllCRDs?

Reply:
Looks like LoadCRDs() is called in a few tests in which the test scenarios are probably for KCC resource (non core/operator ones) only.